### PR TITLE
Fix topaxis bottomhover rangeofthres

### PR DESCRIPTION
**Pull request by lebronlambert**
Fix the bug of top_axis 
Add the bottom hover 
Add the range of threshold
rewrite the readme

### CODE REVIEW

**Comment by blengerich** [comment]
Can one of the admins verify this patch?

**Comment by blengerich** [comment]
How large are the data files? Does it make sense for us to hold these in the repo?

**Comment by YeWenting** [comment]
I have delete the plink example data and merge the master to this branch. I think we could merge this now.

File: src/frontend/webapp.js [hunk 1]
```diff
@@ -296,6 +296,7 @@ app.post(config.api.requestUserConfirmUrl, function (req, res) {
                 + req.body.code + '</b>'
                 + html_2
             };
+            console.log("hello22222222222")
```
Comment (blengerich) [inline src/frontend/webapp.js]: probably should remove

File: src/frontend/webapp.js [hunk 2]
```diff
@@ -281,9 +281,9 @@ app.post(config.api.requestUserConfirmUrl, function (req, res) {
             //var auth_details = require('./static/email2/Authentication.json');
             //if (err) {throw err}
             //var transporter = nodemailer.createTransport(auth_details.user + ':' + auth_details.pass);
-            //var transporter = nodemailer.createTransport('smtps://xiang.liu1995@gmail.com:19951220lx@@smtp.gmail.com');
-            var transporter = nodemailer.createTransport('smtps://genamap.v2.0@gmail.com:GenAMapV2@smtp.gmail.com')
-            //var transporter = nodemailer.createTransport('smtps://email@gmail.com:pass@smtp.gmail.com');
+            //var transporter = nodemailer.createTransport('smtps://genamap.v2.0@gmail.com:GenAMapV2@smtp.gmail.com')
+            //var transporter = nodemailer.createTransport('smtps://wentingye52@gmail.com:1996111ywt@smtp.gmail.com')
+            var transporter = nodemailer.createTransport('smtps://email@gmail.com:pass@smtp.gmail.com');
```
Comment (blengerich) [inline src/frontend/webapp.js]: how does this get the authentication details? We don't want to write them in the repo in plaintext, which is why we previously used Authentication.json (not checked in to the repo).

File: src/frontend/package.json [hunk 3]
```diff
@@ -28,6 +28,7 @@
     "css-loader": "^0.28.4",
     "csvtojson": "^1.1.7",
     "d3": "^4.9.1",
+    "dimensions": "^0.1.0",
```
Comment (blengerich) [inline src/frontend/package.json]: looks like the only place this is used is commented out - is this intended for future incorporation?

Comment (lebronlambert) [inline src/frontend/package.json]: I have removed it, for I once used it , but after all I delete it.

File: src/frontend/webapp.js (hunk 1 above)
Comment (lebronlambert) [inline src/frontend/webapp.js]: done

File: src/frontend/webapp.js (hunk 2 above)
Comment (lebronlambert) [inline src/frontend/webapp.js]: done

Comment (blengerich) [inline src/frontend/webapp.js]: Good to remove the plaintext. How does line 282 get the authentication details?

**Comment by blengerich** [comment]
Please fix the email authentication, then I will merge.

**Pull request merged**